packages/web/src/utils/index.js

```diff
@@ -169,3 +169,30 @@ export const getPopularSuggestionsComponent = (data = {}, props = {}) => {
 	}
 	return null;
 };
+
+// to check whether arrays are equal in values
+// irrespective of the order of entries
+export const areArraysEqual = (arr1, arr2) => {
```

Comment:
You may wish to create a single utilities file shared among libraries instead of copy-pasta and ending up with duplicates.

Reply:
It would be removed because of no usage.